client/client.go

```diff
@@ -480,57 +465,40 @@ func (client *Client) doDial(
 
 	if routingRuleForDomain == domainrouting.MustDirect {
 		log.Debugf("Forcing direct to %v per domain routing rules (MustDirect)", host)
-		op.Set("force_direct", true)
-		op.Set("force_direct_reason", "routingrule")
 		return dialDirect(ctx, "tcp", addr)
 	}
 
 	if shouldForceProxying() {
 		log.Tracef("Proxying to %v because everything is forced to be proxied", addr)
-		op.Set("force_proxied", true)
-		op.Set("force_proxied_reason", "forceproxying")
 		return dialProxied(ctx, "whatever", addr)
 	}
 
 	if routingRuleForDomain == domainrouting.MustProxy {
 		log.Tracef("Proxying to %v per domain routing rules (MustProxy)", addr)
-		op.Set("force_proxied", true)
-		op.Set("force_proxied_reason", "routingrule")
 		return dialProxied(ctx, "whatever", addr)
 	}
 
 	if err := client.allowSendingToProxy(addr); err != nil {
 		log.Debugf("%v, sending directly to %v", err, addr)
-		op.Set("force_direct", true)
-		op.Set("force_direct_reason", err.Error())
 		return dialDirect(ctx, "tcp", addr)
 	}
 
 	if client.proxyAll() {
 		log.Tracef("Proxying to %v because proxyall is enabled", addr)
-		op.Set("force_proxied", true)
-		op.Set("force_proxied_reason", "proxyall")
 		return dialProxied(ctx, "whatever", addr)
 	}
 
 	dialDirectForShortcut := func(ctx context.Context, network, addr string, ip net.IP) (net.Conn, error) {
 		log.Debugf("Use shortcut (dial directly) for %v(%v)", addr, ip)
-		op.Set("shortcut_direct", true)
```

Comment:
why did you remove all the `op`s?

Reply:
Mostly they were annoying me, and I don't think they're being used, but I'll put 'em back =).